src/main/java/com/linecorp/armeria/server/docs/ThriftDocString.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2016 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.docs;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.reflections.Reflections;
+import org.reflections.scanners.ResourcesScanner;
+import org.reflections.util.ClasspathHelper;
+import org.reflections.util.ConfigurationBuilder;
+import org.reflections.util.FilterBuilder;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+
+/**
+ * {@link ThriftDocString} is an DocString extractor for Thrift IDL JSON.
+ * If you want use DocString in {@link DocService}, generate YOUR.json using YOUR.thrift and
```

Comment:
```java
To include docstrings in {@link DocService} pages, use a recent development version of the Thrift compiler (0.9.3 will not work) and compile your thrift files with 'json' code generation and include the resulting json files in the classpath location META-INF/armeria/thrift. The classpath location can be changed by setting the armeria.thrift.json.dir system property.
```

Careful with the word resources/ as it's an IDE/build system concept, not a Java concept.